api/v1alpha1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1alpha1 contains API Schema definitions for the operator v1alpha1 API group
-//+kubebuilder:object:generate=true
-//+groupName=externaldns.olm.openshift.io
+// +kubebuilder:object:generate=true
```

Comment:
Do you need to regenerate something after the api changes?

Reply:
That was just golang 1.19 new formatting of the comment markers, now they need a whitespace after `//`. No need to regenerate anything, if there would be anything we would catch it in the CI: it runs the markers to code and code to manifests generators to see if there is anything not committed.

Reply:
This is already in <https://github.com/openshift/external-dns-operator/pull/179/commits/a935fda8a25498c8098f920074ad8fc4915f6f8f#diff-dab150f060822d42b5ee452169a22a98dca6669fef3b58f23a116caf26084955>.  Presumably this change and the one in `api/v1beta1/groupversion_info.go` would vanish with a rebase.

Reply:
Yes, rebased from `main`.